cypress/tests/integration/stakeholder/[stakeholderid].cy.js

```diff
@@ -0,0 +1,60 @@
+import species from '../../../../doc/examples/species/1.json';
+import stakeholder from '../../../../doc/examples/stakeholders/180Earth.json';
+import planter from '../../../fixtures/940.json';
+import exampleTreeData from '../../../fixtures/tree186734.json';
+import { prepareNocks, clearNocks } from '../nockRoutes';
+
+beforeEach(() => {
+  clearNocks();
+});
+
+describe('Stakeholder', () => {
+  const imageFixturePath = `images/organization.png`;
+  return it(`stakeholder test`, () => {
+    const path = `/stakeholder/${stakeholder.id}`;
+    cy.fixture(imageFixturePath).then((image) => {
+      const blob = Cypress.Blob.base64StringToBlob(image, 'images/png');
+      const photo_url = Cypress.Blob.createObjectURL(blob);
+      prepareNocks({ stakeholder: { ...stakeholder, photo_url } });
+    });
+
+    cy.task('nockIntercept', {
+      hostname: 'https://dev-k8s.treetracker.org',
+      method: 'get',
+      path: '/query/v2/stakeholder/1',
```

Comment:
nit: I would use
```
path: `/query/v2/stakeholder/${stakeholder.id}`,
```
instead of
```
path: '/query/v2/stakeholder/1',
```
to populate the stakeholder id in path dynamically. And this should apply to all other paths in this test file if you are modifying any of them.

Reply:
Hi @yunchipang , thanks for your suggestion! I've just changed it. I agree it will make my code look cleaner.